sql/sem/tree: TestPrettyData failed

sql/sem/tree.TestPrettyData [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_MiscNightlyTestsBazel/20904933?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_MiscNightlyTestsBazel/20904933?buildTab=artifacts#/) on master @ [95e070ea5549abfdf72ca87927d8dfa2acb093ce](https://github.com/cockroachdb/cockroach/commits/95e070ea5549abfdf72ca87927d8dfa2acb093ce):


```
        INTO
        	'gs://acme-co-backup'
        AS OF SYSTEM TIME
        	'-10s'
        WITH
        	revision_history = true, incremental_location = 'gs://acme-co-backup/incrementals'
        
        17:
        -----------------
                   BACKUP
                 DATABASE bank
                     INTO 'gs://acme-co-backup'
        AS OF SYSTEM TIME '-10s'
                     WITH revision_history = true, incremental_location = 'gs://acme-co-backup/incrementals'
        
        160:
        ----------------------------------------------------------------------------------------------------------------------------------------------------------------
        BACKUP DATABASE bank INTO 'gs://acme-co-backup' AS OF SYSTEM TIME '-10s' WITH revision_history = true, incremental_location = 'gs://acme-co-backup/incrementals'
        
        
        
        got:
        // Code generated by TestPretty. DO NOT EDIT.
        // GENERATED FILE DO NOT EDIT
        1:
        -
        BACKUP
        DATABASE
        	bank
        INTO
        	'gs://acme-co-backup'
        AS OF SYSTEM TIME
        	'-10s'
        WITH
        	revision_history = true
        
        17:
        -----------------
                   BACKUP
                 DATABASE bank
                     INTO 'gs://acme-co-backup'
        AS OF SYSTEM TIME '-10s'
                     WITH revision_history = true
        
        101:
        -----------------------------------------------------------------------------------------------------
        BACKUP DATABASE bank INTO 'gs://acme-co-backup' AS OF SYSTEM TIME '-10s' WITH revision_history = true
        
        
--- FAIL: TestPrettyData/align-deindent/backup.sql (0.00s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestPrettyData.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-58157

--- a/pkg/sql/sem/tree/testdata/pretty/backup.align-deindent.golden
+++ b/pkg/sql/sem/tree/testdata/pretty/backup.align-deindent.golden
@@ -10,7 +10,7 @@ INTO
 AS OF SYSTEM TIME
 	'-10s'
 WITH
-	revision_history = true, incremental_location = 'gs://acme-co-backup/incrementals'
+	revision_history = true
 
 17:
 -----------------
@@ -18,10 +18,10 @@ WITH
          DATABASE bank
              INTO 'gs://acme-co-backup'
 AS OF SYSTEM TIME '-10s'
-             WITH revision_history = true, incremental_location = 'gs://acme-co-backup/incrementals'
+             WITH revision_history = true
 
-160:
-----------------------------------------------------------------------------------------------------------------------------------------------------------------
-BACKUP DATABASE bank INTO 'gs://acme-co-backup' AS OF SYSTEM TIME '-10s' WITH revision_history = true, incremental_location = 'gs://acme-co-backup/incrementals'
+101:
+-----------------------------------------------------------------------------------------------------
+BACKUP DATABASE bank INTO 'gs://acme-co-backup' AS OF SYSTEM TIME '-10s' WITH revision_history = true
 
 

--- a/pkg/sql/sem/tree/testdata/pretty/backup.align-only.golden
+++ b/pkg/sql/sem/tree/testdata/pretty/backup.align-only.golden
@@ -10,7 +10,7 @@ INTO
 AS OF SYSTEM TIME
 	'-10s'
 WITH
-	revision_history = true, incremental_location = 'gs://acme-co-backup/incrementals'
+	revision_history = true
 
 17:
 -----------------
@@ -18,10 +18,10 @@ WITH
          DATABASE bank
              INTO 'gs://acme-co-backup'
 AS OF SYSTEM TIME '-10s'
-             WITH revision_history = true, incremental_location = 'gs://acme-co-backup/incrementals'
+             WITH revision_history = true
 
-160:
-----------------------------------------------------------------------------------------------------------------------------------------------------------------
-BACKUP DATABASE bank INTO 'gs://acme-co-backup' AS OF SYSTEM TIME '-10s' WITH revision_history = true, incremental_location = 'gs://acme-co-backup/incrementals'
+101:
+-----------------------------------------------------------------------------------------------------
+BACKUP DATABASE bank INTO 'gs://acme-co-backup' AS OF SYSTEM TIME '-10s' WITH revision_history = true
 
 

--- a/pkg/sql/sem/tree/testdata/pretty/backup.ref.golden
+++ b/pkg/sql/sem/tree/testdata/pretty/backup.ref.golden
@@ -10,10 +10,10 @@ INTO
 AS OF SYSTEM TIME
 	'-10s'
 WITH
-	revision_history = true, incremental_location = 'gs://acme-co-backup/incrementals'
+	revision_history = true
 
-160:
-----------------------------------------------------------------------------------------------------------------------------------------------------------------
-BACKUP DATABASE bank INTO 'gs://acme-co-backup' AS OF SYSTEM TIME '-10s' WITH revision_history = true, incremental_location = 'gs://acme-co-backup/incrementals'
+101:
+-----------------------------------------------------------------------------------------------------
+BACKUP DATABASE bank INTO 'gs://acme-co-backup' AS OF SYSTEM TIME '-10s' WITH revision_history = true
 
 

